actioncable/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "actioncable",
+  "version": "5.0.0-beta4",
```

Comment:
I figured the version push would be done by someone on core separate from the PR, but I can push up rc1 if people are okay with that.